### ISSUE TITLE
C++ 20 Support

### ISSUE THREAD

**Issue by vpa1977** [microsoft/krabsetw#165]
I have tried to compile krabsetw examples using `/std::c++latest' . 
Due to the change in widechar handling, the compilation produces following error:
```
Error	C2280	'std::basic_ostream<char,std::char_traits<char>> &std::operator <<<std::char_traits<char>>(std::basic_ostream<char,std::char_traits<char>> &,const wchar_t *)': attempting to reference a deleted function	NativeExamples	.\krabsetw\krabs\krabs\provider.hpp	539	
```

Would It be possible to modify provider name logging to convert it from widechar to utf8 to support C++ 20 ?

**Comment by swannman** [comment]
Good catch @vpa1977! If you'd like to submit a PR to address this I'd be happy to review.

**Comment by vpa1977** [comment]
Thanks !!!
I've added https://github.com/microsoft/krabsetw/pull/166

**Comment by swannman** [comment]
Resolved by #166. Thanks @vpa1977!
